lib/resources/example_json/soarian_examples_conformance.rb

```diff
@@ -0,0 +1,323 @@
+module Cerner
+  module Resources
+
+    SOARIAN_METADATA ||= {
+    "resourceType": "Conformance",
+    "publisher": "Cerner Corporation",
+    "date": "2017-03-15T11:54:23-04:00",
+    "copyright": "Copyright © Cerner Health Services, Inc. All rights reserved. This document contains confidential and/or proprietary information belonging to Cerner Health Services, Inc. and/or its related affiliates (Cerner) which may not be reproduced or transmitted in any form or by any means without the express written consent of Cerner. The Applications may contain embedded third party software including free and open source software (FOSS). Soarian's FOSS manual provides details and limitations that apply.",
+    "kind": "capability",
+    "software": {
+        "name": "Soarian Clinicals CCDS API",
+        "version": "01.00.0000"
+    },
+    "fhirVersion": "HAPI-1.3, FHIR DSTU2 - 1.0.2",
```

Comment:
Amaresh to open JIRA to change fhirVersion to "1.0.2"

Soarian does not have a public sandbox or non-prod endpoint suitable for public consumption; evidently it was on the backlog at one time but not stood up due to resource constraints.  @daliboz , what do you suggest as a "dummy" URL?  Agree we don't want to give the impression that there is a sandbox where none exists.